Makefile.PL

```diff
@@ -13,6 +13,7 @@ WriteMakefile(
                           'PGObject::Simple'                => 0,
                           'PGObject::Simple::Role'          => 0,
                           'PGObject::Util::DBMethod'        => 0,
+                          'PGObject::Util::LogRep::TestDecoding' => 0,
```

Comment:
I did not re-intend the Makefile.PL here because I am concerned about repeated conflicts leading to things getting lost.   Though we have tests for this.  I think we should do another re-indent when there are fewer outstanding PRs